### ISSUE TITLE
Smart-episode ignores repacks

### ISSUE THREAD

**Issue by pyrodex** [autodl-community/autodl-irssi#134]
With "smart-episode = true" and if a repack is released for the show it is ignored and autodl-irssi states the release is already been downloaded.

**Comment by thebigmunch** [comment]
Do you have ``download-duplicates`` set to ``true``?

**Comment by pyrodex** [comment]
No I do not, isn't that kind of against the purpose of smart-episodes?

**Comment by thebigmunch** [comment]
Well, not conceptually. That being said, I don't believe the logic as currently written allows for that interaction. Pretty sure I left it out purposely to keep things simple as it is somewhat uncommon and easy to grab a repack manually. It may not actually be feasible or possible to get it perfect. I was asking for some real world confirmation before looking at the code again, as that would likely be the method to do it if desired.

**Comment by thebigmunch** [comment]
So, I came back to this with fresh eyes. I **think** I've got this fixed up to properly consider repacks/propers as separate releases when using ``smart-episode``. Overwrite the files from [this file](https://waa.ai/zBZz.zip) and test it out. It includes other changes I've made prior to this one as I couldn't avoid that. But they shouldn't cause any trouble or affect this issue; they'll be in the next release anyway.
